code/modules/power/supermatter/crystal_matrix.dm

```diff
@@ -0,0 +1,176 @@
+GLOBAL_LIST_EMPTY(crystal_matrix)
+
+/obj/destabilized_supermatter
+	name = "destabilized crystal"
+	desc = "A strangely translucent and iridescent crystal."
+	icon = 'icons/obj/supermatter.dmi'
+	icon_state = "psy"
+	density = TRUE
+	anchored = TRUE
+	flags_1 = PREVENT_CONTENTS_EXPLOSION_1
+	light_range = 4
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
+
+/obj/destabilized_supermatter/Initialize()
+	. = ..()
+	end_world_activation()
+	SSshuttle.registerHostileEnvironment(src)
+
+/obj/destabilized_supermatter/Destroy()
+	return..()
+
+///Start the process where the crystal matrix expands to devour the world
+/obj/destabilized_supermatter/proc/end_world_activation()
+	priority_announce("WARNING - Possible TK-end of the world scenario approaching, please standby")
+	sound_to_playing_players('sound/misc/notice2.ogg')
+	sleep(0.7 MINUTES)
+	for(var/mob/M in GLOB.player_list)
+		SEND_SOUND(M, 'sound/items/poster_ripped.ogg')
+		sleep(10)
+		SEND_SOUND(M, 'sound/items/poster_ripped.ogg')
+		sleep(7)
+		SEND_SOUND(M, 'sound/items/poster_ripped.ogg')
+		sleep(15)
+		SEND_SOUND(M, 'sound/effects/explosion3.ogg')
+	var/turf/T = get_turf(src)
+	explosion(T,0,5,10,1,1,1)
+	T.ChangeTurf(/turf/closed/indestructible/crystal_matrix_core)
+	priority_announce("WARNING - The crystal matrix has been uncontained! The spread of the free crystal structure is unkown but it has been \
+						calculated to be able to devour the entire region of space. Emergency shuttle can't reach the station, good luck")
+	sound_to_playing_players('sound/machines/alarm.ogg')
+	sleep(2.5 MINUTES)
+	total_annihilation()
+
+///Process the world ending scenario by killing everyone and making everything a crystal matrix
+/obj/destabilized_supermatter/proc/total_annihilation()
+	for(var/T in GLOB.crystal_matrix)
+		var/turf/Turf = T
+		Turf.Melt()
+	priority_announce("The Crystal Matrix has reached the Expansion point! This is a TK-Z Level of End of the World Scenario! \
```

Comment:
yeah, we'll fuckin' do that with the emergency shuttle that you won't give us.

pods and lavaland shuttles won't help much, since lavaland is in our sector of space.